nmigen_boards/orangecrab_r0d2.py

```diff
@@ -0,0 +1,114 @@
+import os
+import subprocess
+import shutil
+
+from nmigen.build import *
+from nmigen.vendor.lattice_ecp5 import *
+from .resources import *
+
+
+__all__ = ["OrangeCrabR0D2Platform"]
+
+
+class OrangeCrabR0D2Platform(LatticeECP5Platform):
```

Comment:
I'd probably use `R0_2`, not `R0D2` (I was wondering: "revision 0, sure, but what is D2?").

Reply:
"decimal" probably, I stole the naming from LUNA